prow/istio-presubmit.sh

```diff
@@ -90,65 +92,23 @@ else
   # Use the current commit.
   GIT_SHA="$(git rev-parse --verify HEAD)"
 fi
-cd $ROOT
-
-# ./bin/generate-protos.sh || die "Could not generate *.pb.go"
-# if [[ -n $(git status --porcelain) ]]; then
-#     git status
-#     die "Repo has unstaged changes. Re-run ./bin/generate-protos.sh"
-# fi
-
-mkdir -p ~/envoy
-cd ~/envoy
-ISTIO_PROXY_BUCKET=$(sed 's/ = /=/' <<< $( awk '/ISTIO_PROXY_BUCKET =/' $ROOT/WORKSPACE))
-PROXYVERSION=$(sed 's/[^"]*"\([^"]*\)".*/\1/' <<<  $ISTIO_PROXY_BUCKET)
-PROXY=debug-$PROXYVERSION
-wget -qO- https://storage.googleapis.com/istio-build/proxy/envoy-$PROXY.tar.gz | tar xvz
-ln -sf ~/envoy/usr/local/bin/envoy $ROOT/pilot/proxy/envoy/envoy
-cd $ROOT
-
-# go test execution
-# time dep ensure -v
-
-# echo FIXME remove mixer tools exclusion after tests can be run without bazel
-# time go test $(go list ./mixer/... | grep -v /tools/codegen)
-# time go test ./pilot/...
-# time go test ./security/...
-# time go test ./broker/...
-# rm -rf vendor/
 
-if [ "${CI:-}" == 'bootstrap' ]; then
-  # Test harness will checkout code to directory $GOPATH/src/github.com/istio/istio
-  # but we depend on being at path $GOPATH/src/istio.io/istio for imports
-  mv ${GOPATH}/src/istio.io/istio ${GOPATH}/src/github.com
-  ln -sf ${GOPATH}/src/github.com/istio ${GOPATH}/src/istio.io
-  ROOT=${GOPATH}/src/istio.io/istio
-  cd ${GOPATH}/src/istio.io/istio
-fi
+echo 'Pulling down pre-built Envoy'
+fetch_envoy
 
-# Build
+echo 'Building everything'
 ${ROOT}/bin/init.sh
 
-#run_or_die_on_change ./bin/fmt.sh
-
-# bazel test execution
 echo 'Running Unit Tests'
 time bazel test --test_output=all //...
 
-# run linters in advisory mode
+echo 'Running linters in advisory mode'
```

Comment:
what does that mean by "advisory mode"?

Maybe that meant that linters didn't cause presubmit to fail, but it isn't true anymore since presubmit now fails when linters report lint warnings. Let's remove the "advisory mode" if so.